.ci/jobs.t/elastic+elasticsearch+periodic+dra-snapshots.yml

```diff
@@ -56,9 +60,9 @@
                    docker.elastic.co/infra/release-manager:latest \
                    cli collect \
                    --project elasticsearch \
-                   --branch "master" \
+                   --branch "$RM_BRANCH" \
```

Comment:
I think we're missing the bit that actually defines this env var.

Reply:
fixed